R/pkg/R/DataFrame.R

```diff
@@ -2514,7 +2529,9 @@ setMethod("attach",
 #' environment. Then, the given expression is evaluated in this new
 #' environment.
 #'
+#' @title with
```

Comment:
@shivaram Yes, I also notice titles of other examples are not consistent. Which one should we use? Short description or just the name of the method.